rfcs/text/0001-wildcard-data-type.md

```diff
@@ -115,56 +254,70 @@ Stage 2: Identifies scope of impact of changes. Are breaking changes required? S
 The goal here is to research and understand the impact of these changes on users in the community and development teams across Elastic. 2-5 sentences each.
 -->
 
-ECS-compliant data consumers should not be impacted due to the `keyword` field family[2], however ECS-compliant producers will be required for their field mappings to use `wildcard` where defined by the spec. The ECS project will need to evaluate how best to
-evolve its tooling to support `wildcard`; wildcard will be the first time a Elastic licensed feature is specified in ECS.
-
 ### Ingestion
 
-Any component producing data (Beats, Logstash, third-party developed, etc.) will need to comply with the new mappings.
+Any component producing data (Beats, Logstash, third-party developed, etc.) will need to adopt the mappings in their index templates. The discussion around the handling of OSS vs. Basic field types between OSS and Basic modules/plugins will be handled outside of this proposal.
 
```

Comment:
Another important item outside the scope of this proposal is backwards compatibility.  Producers will need to consider the case where the producer has adopted wildcard, but the Elasticsearch instance they are sending data to doesn't support wildcard (example ES 6.8).